contracts/UniswapV3Staker.sol

```diff
@@ -266,6 +266,22 @@ contract UniswapV3Staker is IUniswapV3Staker, Multicall {
         emit RewardClaimed(to, reward);
     }
 
+    function claimRewardAmount(
+        IERC20Minimal rewardToken,
+        address to,
+        uint256 amountRequested
+    ) external override returns (uint256 reward) {
```

Comment:
does it seem inconsistent to return a reward amount here but not on `claimReward`?

Reply:
About to remove `claimReward` and `claimRewardAmount` -> `claimReward`. Do you think it should have a return value or no?

Reply:
I'm gonna drop it

Reply:
not sure. 

I think the return val it's only useful for external contracts calling in, and they would've had to pass in the rewardAmount as calldata here.  For `claimReward`, they could get rewardaAmount from the view function. So maybe leaning towards removing, but:

Interested to see what @moodysalem thinks??

Reply:
just got your drop it comment. works 4 me!